j.java

```diff
@@ -0,0 +1,11 @@
+package com.iluwatar.callback;
+
+/**
+ * 
+ * Callback interface
+ * 
+ */
+public interface Callback {
+    // i hote java with a pasion tokuns gety weihgt lenhgt
```

Comment:
Typo inside comment in token 'hote'. Possible corrections:note (72),hot (62),home (10)